.github/actions/docker-deploy/action.yml

```diff
@@ -0,0 +1,172 @@
+name: Deploy Docker Image to Cloud Run
+description: Deploy a docker image using Cloud Run.  This will override any deployment with the same prefix.  Returns the URL of the deployed image as `deployed-url`.
+inputs:
+  docker-registry-name:
+    required: true
+    description: GCP Docker Registry name
+  gcp-provider-id:
+    required: true
+    description: CAS GCP Provider ID
+  gcp-service-account-email:
```

Comment:
This is an extreme nitpick, but it's weird to me that this is "service account email" and deployment-service-account is just "service account".  I also think that the names+descriptions for those two inputs don't super clearly explain the difference between them.

Reply:
Fait enough!  Lemme take another stab at it